docs/mimimal-softlayer.yml

```diff
@@ -366,6 +307,62 @@ jobs:
   properties:
     login:
       catalina_opts: -Xmx768m -XX:MaxPermSize=256m
+      saml:
+        serviceProviderCertificate: |
+          -----BEGIN CERTIFICATE-----
+          MIIENjCCAh6gAwIBAgIQKKBo+285koaiAaJKVNstEzANBgkqhkiG9w0BAQsFADAZ
```

Comment:
OK, @petergtz but clearly these certificate should not be certificates used in any parts of your testing or other envs. Goes w/o saying and just confirming. Thx